src/sentry/sentry_metrics/metrics_interface.py

```diff
@@ -0,0 +1,38 @@
+from typing import Mapping
+
+from sentry.sentry_metrics.use_case_id_registry import UseCaseID
+
+
+class GenericMetricsBackend:
```

Comment:
I'm assuming `GenericMetricsBackend` should never be instantiated? If that's the case we should inherit from [`ABC`](https://docs.python.org/3/library/abc.html), and mark the methods as `@abstractmethod`. But I could be miss-understanding this, please correct me if I'm wrong.

Reply:
I think you're right, thanks for pointing it out